Packs/Base/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -0,0 +1,585 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import urllib
+from typing import List, Dict, Tuple
+import pandas as pd
+import base64
+import requests
+import dill
+import copy
+import numpy as np
+from tldextract import TLDExtract
+requests.packages.urllib3.disable_warnings()
+
+dill.settings['recurse'] = True
+
+NO_FETCH_EXTRACT = TLDExtract(suffix_list_urls=None, cache_dir=False)
+
+OOB_MAJOR_VERSION_INFO_KEY = 'major'
+OOB_MINOR_VERSION_INFO_KEY = 'minor'
+MAJOR_VERSION = 0
+MINOR_DEFAULT_VERSION = 0
+
+MSG_INVALID_URL = "Error: "
+MSG_NO_URL_GIVEN = "Please input at least one URL"
+MSG_FAILED_RASTERIZE = "Rasterize for this url did not work correctly"
+MSG_IMPOSSIBLE_CONNECTION = "Failed to establish a new connection - Name or service not known"
+MSG_UPDATE_MODEL = "Update demisto model from docker model version %s.%s"
+MSG_UPDATE_LOGO = "Update demisto model from docker model version %s.%s and transfering logos from demisto version %s.%s"
+MSG_WRONG_CONFIG_MODEL = 'Wrong configuration of the model'
+MSG_NO_ACTION_ON_MODEL = "Use current model"
+MSG_WHITE_LIST = "White List"
+EMPTY_STRING = ""
+URL_PHISHING_MODEL_NAME = "phishing_model"
```

Comment:
It's our default name for the phishing model so it's not good.
Please change it to "url_phishing_model"
Also, set it as automation argument with a default